client/logging/src/lib.rs

```diff
@@ -269,10 +272,10 @@ mod tests {
 		assert!(output.contains(EXPECTED_LOG_MESSAGE));
 	}
 
-	/// This is no actual test, it will be used by the `dash_in_target_name_works` test.
-	/// The given test will call the test executable to only execute this test that
-	/// will only print `EXPECTED_LOG_MESSAGE` through logging while using a target
-	/// name that contains a dash. This ensures that targets names with dashes work.
+	/// This is not an actual test, it is used by the `dash_in_target_name_works` test.
+	/// The given test will call the test executable and only execute this one test that
+	/// only prints `EXPECTED_LOG_MESSAGE` through logging while using a target
+	/// name that contains a dash. This ensures that target names with dashes work.
```

Comment:
There a bunch of more copies of this comment. Mind fixing those too?

Reply:
Fixed in 75672d9f8

Maybe double check because I'm not a native english speaker.

Also: who cares as long as people understand? It's not even doc

Reply:
@cecton happy to do a tech doc edit if helpful, just ping me on any issues you want double checked :two_women_holding_hands:

Reply:
OMG! That is awesome! Yes of course, feel free to directly edit on this branch/PR if you want. I will solve any conflict